cpp/include/raft/sparse/distance/distance.cuh

```diff
@@ -130,8 +132,101 @@ void pairwiseDistance(value_t* out,
   }
 }
 
-};  // namespace distance
-};  // namespace sparse
-};  // namespace raft
+/**
+ * @defgroup sparse_distance Sparse Pairwise Distance
+ * @{
+ */
+
+/**
+ * @brief Compute pairwise distances between x and y, using the provided
+ * input configuration and distance function.
+ *
+ * @code{.cpp}
+ * #include <raft/core/device_resources.hpp>
+ * #include <raft/core/device_coo_matrix.hpp>
```

Comment:
Do we want coo or csr matrix here? You are calling `make_device_csr_matrix` below.